dashboard/app/controllers/api/v1/pd/applications_controller.rb

```diff
@@ -203,18 +203,18 @@ def get_applications_by_role(role, include_associations: true)
       applications_of_type = @applications.where(type: TYPES_BY_ROLE[role].try(&:name))
       applications_of_type = applications_of_type.includes(:user, :regional_partner) if include_associations
       case role
-        when :csf_facilitators
-          return applications_of_type.csf
-        when :csd_facilitators
-          return applications_of_type.csd
-        when :csp_facilitators
-          return applications_of_type.csp
-        when :csd_teachers
-          return applications_of_type.csd
-        when :csp_teachers
-          return applications_of_type.csp
-        else
-          raise ActiveRecord::RecordNotFound
+      when :csf_facilitators
```

Comment:
We could also do
`applications_of_type.send(role.to_s.split('_').first.to_sym)` if we want to avoid the case. But that may be aggressive DRY